storage/bootstrap.go

```diff
@@ -36,16 +36,14 @@ import (
 
 type bootstrapState int
 
+// nolint: deadcode
```

Comment:
This one is only for the statement block. Good idea, I'll write up a quick doc for it.